styles/signals.json

```diff
@@ -25,6 +25,7 @@
 				"coordinates": [[10,50],[80,50]],
 				"properties": {
 					"railway":"rail",
+					"railway:atb":"no",
```

Comment:
added this line, I guess it's incorrect otherwise

Reply:
it only needs to match on one of the "nothing" conditions, either lzb/pzb/etcs or atb/etcs. It's not wrong this way, just not needed. But it doesn't hurt either.